src/gafaelfawr/cli.py

```diff
@@ -93,11 +93,12 @@ def generate_token() -> None:
     default=None,
     help="Application settings file.",
 )
-def init(settings: Optional[str]) -> None:
+@coroutine
```

Comment:
This decorator is pretty cool! It is something else to port to Safir?

Reply:
Yeah, if you have another package that wants to use it, I'd be happy to move it to Safir. I wasn't sure how common the use case of a CLI would be.